fields_service/views/fields_views.py

```diff
@@ -0,0 +1,94 @@
+"""Creates resources"""
+from flask_restful import Resource
+from flask import request, Response
+from fields_service.models.field import Field
+from fields_service.models.choice import Choice
+from fields_service.serializers.field_schema import FieldSchema
+from fields_service.db import DB
+
+
+#pylint: disable = R0201
```

Comment:
write in disabled names of warning instead of code, please
if it is no-member add to .pylintrc to generated members: add, commit, delete, query, rollback